client/helpers/render-elements.helpers.tsx

```diff
@@ -202,23 +214,24 @@ const renderSelectBoxes = (e: any, data: any, container: string) => {
   );
 };
 
-const renderSignature = (e: any, data: any, container: string) => {
+const renderSignature = (e: any, data: any) => {
   const label = getLabel(e);
+  const value = getValue(e.key, data);
 
   return (
     <div key={e.id} className='w-1/2 grid grid-flow-row'>
       <span className='font-bold'>{label}</span>
-      <Image src={data[container][e.key]} alt={e.label} width='50' height='100' />
+      <Image src={value} alt={label} width='50' height='100' />
     </div>
   );
 };
 
-const renderUsageCountForm = (e: any, data: any, container: string) => {
+const renderUsageCountForm = (e: any, data: any) => {
```

Comment:
are we adding a check to hide the table if the form is not applicable to the application?

Reply:
I can add that 👍🏻